client/www/pages/docs/strong-init.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Strong Init (experimental)
+---
+
+We're piloting strictly-typed variants of our client libraries that leverage your `instant.schema.ts` definition. We're calling it "Strong Init". Strong Init enhances InstaQL and InstaML methods with deep typesafety and enhances your editor experience with rich intellisence.
+
+If you want to migrate, simply swap out `init` with `init_experimental` and provide it your app's graph schema object.
+
+```ts
+import { init_experimental } from '@instantdb/react';
+import schema from '~/instant.schema.ts';
+
+const db = init_experimental({
+  appId: '__APP_ID__',
+  schema,
+});
+
+function App() {
+  return <Main />;
+}
+```
+
+## Strong InstaML
```

Comment:
I see the example for InstaML, but not for useQuery. 

Maybe we integrate this in the code example above, so there's an example with `useQuery` that is typed, as well as db.tx